backend/census_historical_migration/historic_data_loader.py

```diff
@@ -0,0 +1,30 @@
+from .models import ELECAUDITHEADER as Gen
+from .make_submission import load_historic_data
+
+
+def load_historic_data_for_year(audit_year):
+    """Iterates over and processes submissions for the given audit year"""
+    result_log = {}
+    total_count = error_count = 0
+
+    gens = Gen.objects.filter(AUDITYEAR=audit_year)
+    for gen in gens:
+        dbkey = gen.DBKEY
+        result = load_historic_data(audit_year, dbkey)
```

Comment:
We may want to add a comment here since it is the entry point for the process. So, the rest of the code ignore the audit_year parameter because it is assumed that we do migration year-by-year. That is at no point there will be audit data for more than a year in the historic database. Let's capture this here. We should probably have something in place to ensure this assumption remains true before each migration cycle (this last part is not necessary related to your PR).

Reply:
> Per conversation, we decided that we'll thread the year through, because loading historical data year-by-year (e.g. remove 2022, then load 2021...) would be annoying in production. It might be best to assume all of the historical data is loaded, and we select what we need from it. Whether that is for this PR or a future was left for the group to decide as to what is best.

Agree!

Reply:
This [ticket ](https://github.com/GSA-TTS/FAC/issues/2879) is to revisit the entire migration logic and enforce audit_year in all queries.

Reply:
Renamed `gens`/`gen` and added a comment